melosys-eessi-app/src/main/java/no/nav/melosys/eessi/identifisering/BehandleBucIdentifisertService.java

```diff
@@ -30,7 +32,28 @@ public void bucIdentifisert(String rinaSaksnummer, String aktoerId) {
         sedMottattHendelseRepository.findAllByRinaSaksnummerAndPublisertKafkaSortedByMottattDato(rinaSaksnummer, false)
             .stream()
             .filter(sedMottattHendelse -> sedMottattHendelse.getSedHendelse().erIkkeX100())
-            .forEach(sedMottattHendelse -> publiserMelding(sedMottattHendelse, aktoerId));
+            .forEach(sedMottattHendelse -> sedIdentifisert(sedMottattHendelse, aktoerId));
```

Comment:
Klassen er en sentral komponent som inntil nå bare ble testet gjennom komponenttesting (publisering på topic).
Opprettelse av journalposter som trengs bør også testes nå, enten med enhetstest eller komponenttest.

Reply:
Laget test for BehandleBucIdentifisertService. Tester i hovedsak journalføring og sending av melding til kafka/melosys-api for identifisert BUC med flere SEDer